examples/using-gatsby-with-json-yaml/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
This file needs to be updated to fit into the examples repo: that means removing all extra fields. Here is an example: https://github.com/gatsbyjs/gatsby/blob/master/examples/using-medium/package.json